src/main/asciidoc/_chapters/upgrading.adoc

```diff
@@ -799,6 +806,21 @@ To upgrade an existing HBase 1.x cluster, you should:
 * Upgrade RegionServers
 * (Eventually) Upgrade Clients
 
+[[upgrade1.7.x]]
+=== Upgrading to 1.7.1+
+HBase release 1.7.0 introduced an incompatible table metadata serialization format that broke the
+minor release compatibility guarantees. The issue was reported in https://issues.apache.org/jira/browse/HBASE-26021[HBASE-26021] and the problematic serialization patch was reverted in
+HBase 1.7.1. Some important notes about 1.7.x upgrades below.
+
+* If you are considering an upgrade to 1.7.x version, skip 1.7.0 completely and upgrade to 1.7.1+
+version. 1.7.0 was withdrawn and removed from the Apache sites.
+* If you already installed a 1.7.0 cluster from scratch and are looking to migrate to 1.7.1+, you
```

Comment:
What if I upgraded from 1.6 to 1.7.0 and am now running 1.7.0?

Reply:
You cannot, the bug doesn't let you upgrade. During the upgrade, 1.7.0 binaries try to parse the table descriptors and it blows up (META to begin with).